client/rest-high-level/src/test/java/org/elasticsearch/client/security/ClearRolesCacheResponseTests.java

```diff
@@ -54,7 +54,9 @@ public void testParseFromXContent() throws IOException {
             XContentParser parser = JsonXContent.jsonXContent.createParser(
                 NamedXContentRegistry.EMPTY,
                 DeprecationHandler.THROW_UNSUPPORTED_OPERATION,
-                "{" + nodesHeader + clusterName + "\"nodes\" : { \"id1\": { \"name\": \"a\"}, \"id2\": { \"name\": \"b\"}}}"
+                """
+                    {%s%s"nodes" : { "id1": { "name": "a"}, "id2": { "name": "b"}}}
```

Comment:
Same here, we can take out the trailing comma from `nodesHeader` and `clusterName` and make this more readable.